packages/zipkin-instrumentation-hapi/test/integrationTest.js

```diff
@@ -0,0 +1,152 @@
+const sinon = require('sinon');
+const {Tracer, ExplicitContext} = require('zipkin');
+const Hapi = require('hapi');
+const middleware = require('../src/hapiMiddleware');
+
+describe('hapi middleware - integration test', () => {
+  it.skip('should create traceId', () => {});
```

Comment:
I copied the tests from `zipkin-instrumentation-express`, which included this pending one. Should I remove it?